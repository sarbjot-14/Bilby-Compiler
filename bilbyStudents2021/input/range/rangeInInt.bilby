main{
	
    imm result := 2 in <4 .. 20> ;
    print result $n;
    
    imm result1 := 4 in <4 .. 20> ;
    print result1 $n;
    
    
    imm result2 := 6 in <4 .. 20> ;
    print result2 $n;
    
    
    imm result3 := 20 in <4 .. 20> ;
    print result3 $n;
    
    imm result4 := 25 in <4 .. 20> ;
    print result4 $n;
    
	
}