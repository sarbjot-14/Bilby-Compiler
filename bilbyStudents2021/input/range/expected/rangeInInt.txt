[ 174 2 4 179 20 ]
false
[ 174 175 4 4 187 20 ]
true
[ 174 176 6 4 195 20 ]
true
[ 174 177 20 4 203 20 ]
true
[ 174 178 25 4 211 20 ]
false
